### ISSUE TITLE
Access to repository

### ISSUE THREAD

**Issue by mpopke** [DallasMuseumArt/FriendsRecommendations#2]
I have a new branch where I've started making changes to Recommendations to enable it to use partials instead of views. I was going to push it up alongside master for general comment (since I know Carlos is planning to do a significant amount of work on this plugin), but I don't appear to have write access to the repository. I can fork it and pull request, but since I'm not working on master right now that seems a little excessive. Is there a specific reason why this repo is private in the first place? I'm not seeing anything particularly sensitive there, but I haven't gone through the whole repo.

**Comment by karnold** [comment]
there is no reason for it to be private. I'm out of town and will give you access when I have a chance to be at my computer

—
Kristen

**Comment by carlos-arroyo** [comment]
Hi Matt, It seems that the repository is public not sure if Kristen has already change it. I am out town as well.  I had added to you permissions access to this repository.

carlos

**Comment by mpopke** [comment]
Thanks, guys. I've pushed a branch up that makes use of partials, and can use the same partial as the ActivityCatalog component I added to the Friends plugin. I'm working right now on extending the recommendations component so that it supports the same filters that the catalog component currently does. Then I'll try to generalize the markup produced by the catalog and filters as much as possible. It's a little DAM-specific right now because I have another demo tomorrow that got sprung on me late last week.
